utils/migrationutils/migrator_test.go

```diff
@@ -2,13 +2,15 @@ package migrationutils
 
 import (
 	"github.com/stretchr/testify/require"
+	"io/ioutil"
 	"os"
 	"path"
 	"testing"
 )
 
 func TestMigrate(t *testing.T) {
-	tmpPath := path.Join(t.TempDir(), "xxx")
+	tmpPath, err := ioutil.TempDir("", "xxx")
```

Comment:
This will return the OS's default dir for temporary files (see `os.TempDir`)

Use `t.TempDir()` instead, which will ensure that whatever you do is cleaned and won't be conflicted with other tests